util/chplenv/chpl_3p_libunwind_configs.py

```diff
@@ -16,7 +16,25 @@ def get_uniq_cfg_path():
 @memoize
 def get_link_args(unwind):
     platform_val = chpl_platform.get('target')
-    linux = platform_val.startswith('linux64')
-    if (unwind == 'libunwind' or unwind == 'system') and linux:
-        return ['-lunwind']
-    return []
+    osx = platform_val.startswith('darwin')
+
+    # Mac OS X supports libunwind in the C library
+    # it's not actually a special library.
+    if osx:
+      return []
+
+    libs = []
+    # Get the link arguments (e.g. -lunwind)
+    if unwind == 'system':
+      # Try using pkg-config to get the libraries to link
+      # libunwind with.
+      libs = third_party_utils.pkgconfig_get_link_args(
```

Comment:
Does the system install not have the same problem with the .pc file not including -lzma?